scripts/format_elevator_data.exs

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env -S ERL_FLAGS=+B elixir
+
+# Script used to format the spreadsheet found at
+# https://docs.google.com/spreadsheets/d/1lHogme-2SuDSgjrRK52k7yVgSFK-v_LMmUfkYgBJjIU/edit?gid=179933470#gid=179933470
```

Comment:
It occurs to me that this spreadsheet probably doesn't need to be public to the world, as the original copy is, given we are not programmatically downloading it. But we'd have to figure out what we're doing about access control on the team.